validphys2/src/validphys/dataplots.py

```diff
@@ -1159,7 +1155,7 @@ def plot_xq2(
 
     """
 
-    w,h = plt.rcParams["figure.figsize"]
+    w,h = mpl.rcParams["figure.figsize"]
```

Comment:
This `rcParams` are another global, I wonder if there's a better way (or even just a way to make them more local...).

Reply:
Ciao @AleCandido, thank you for the comment!